changelogs/fragments/56567-Fix-win_iis_website-id assignment-for-first-new-site.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - win_iis_website - "Fixed #47057 , site_id not used if no sites exist already and creating a new site"
```

Comment:
Unless you quote the entire line, `#` will comment out the rest of the line, this will result in the following data structure:

```
{
    "bugfixes": [
        "win_iis_website - \"Fixed "
    ]
}
```

I would include the full URL to the issue/pr in the changelog fragment.